airbrakes/hardware/imu.py

```diff
@@ -103,16 +113,18 @@ def get_imu_data_packet(self) -> IMUDataPacket | None:
     def get_imu_data_packets(self) -> collections.deque[IMUDataPacket]:
         """
         Returns all available data packets from the IMU.
-        :return: A deque containing the specified number of data packets
+        :return: A deque containing the latest data packets from the IMU.
         """
         # We use a deque because it's faster than a list for popping from the left
-        data_packets = collections.deque()
-        # While there is data in the queue, get the data packet and add it to the dequeue which we
-        # return
-        while not self._data_queue.empty():
-            data_packets.append(self.get_imu_data_packet())
-
-        return data_packets
+        try:
+            return collections.deque(
+                self._data_queue.get_many(block=False, max_messages_to_get=MAX_FETCHED_PACKETS)
+            )
+            # If we had done block=True, then Ctrl+C would not have worked, so we use block=False
+            # to allow for KeyboardInterrupts. The library does not seem to handle
+            # KeyboardInterrupts well, so we have to watch that carefully.
+        except Empty:  # If the queue is empty, don't bother waiting.
+            return collections.deque()
```

Comment:
we should probably do `block=True` and wait for a max of 1 second because I don't want high CPU usage for no reason. Needs to be again tested with the Pi.